nas/handler.go

```diff
@@ -36,28 +36,30 @@ func HandleNAS(ue *context.RanUe, procedureCode int64, nasPdu []byte) {
 		} else {
 			if amfSelf.EnableSctpLb {
```

Comment:
Is there a case where `amfSelf.EnableSctpLb == true` and `amfSelf.EnableDbStore == false` that we should address?

Reply:
No need to handle this scanario